### DISCUSSION TITLE
【1.0.0 ドックフーディング】手順、スクリーンショットについて

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#149] [【運営】相談・報告]
Discordの情報どこかいかないようにこちらにまとめをば....！

**Comment by pss-aileen** [comment]
## スクリーンショット撮影ルール
- ライトモードでの撮影（特にブラウザ、GitHubWeb上の操作）
- ブラウザごとにまるっとスクショをとる

## 画像の共有
- Figmaにて

**Comment by kazzyfrog** [comment]
ポイント？としては、
開発者ではなく、ユーザーの目線になってやってみること！

> これは誰のためのものですか？
> - Git/GitHub の、実践的な使い方を学びたい初心者
> - オープンソースに興味があるが、どのように貢献すればよいかわからない人々

その上で、とりあえずは自分たちが、違和感なくできる感じのラインに持っていく！

**Reply by kazzyfrog** [reply]
メンテナーは、レビュー・承認・コンフリクトが発生した時の解消、、などについても確認！
